### PR TITLE
Feedback survey migration

### LINKED ISSUE #488: Move the survey in-app
Similar to the repot a problem set-up, "Feedback" drawer at the bottom of the page using the email backend.

Edit: implemented as a drawer right now, but it's too long and too narrow. I think a modal might be better. 

~While you're at it, set up the email backend with a MongoDB to store valid submissions in. Wasn't important to the report a problem forms so I left it a TODO at the time, but we're going to want the feedback data to all exist in one easily queryable location.~ Edit: This part's done! The feedback template is also all finished in the email_backend, it's mostly just putting together a better front end for the survey and lining everything else up (remove other survey links, update the survey campaign popup, coordinate with Comms, etc.)

### PR DESCRIPTION
**Pull request by bill-lliu**
moving the survey to be on our mongo server instead of as a redirecting hyperlink
This pull request is only the frontend - backend will be another pull request setting up the mongodb communication
fixes #488

### CODE REVIEW

File: client/src/components/PageDetails.js [hunk 1]
```diff
@@ -43,6 +43,13 @@ export class PageDetails extends React.Component {
             persist_content={true}
           />
         </div>
+        <div className="pagedetails__report-a-problem fcol-md-8 fcol-sm-12">
+          <Details
+            summary_content={text_maker("feedback")}
+            content={<EmailFrontend template_name="feedback" />}
+            persist_content={true}
+          />
+        </div>
```
Comment (Stephen-ONeil) [inline client/src/components/PageDetails.js]: Remember to disable this before merging, we'll stick with the legacy survey page until after the current popup cmpaign ends and the mongo logging's ready.

Comment (bill-lliu) [reply client/src/components/PageDetails.js]: added a radio buttons case to test template 👍

Comment (bill-lliu) [reply client/src/components/PageDetails.js]: and updated for the end to end testing too 👍

**Comment by bill-lliu** [comment]
I'll need to make a second version of this when I add the Mongo for the Feedback survey as well; 
I could make another branch for that, or just to push the `email-mongo` branch first, rebase this `email-backend-update` branch, then add more commits

File: email_backend/templates/feedback.json [hunk 2]
```diff
@@ -39,7 +39,7 @@
       "form_type": "radio",
       "form_label": {
         "en": "To what extent did you find GC InfoBase easy to navigate? (1=not at all useful and 5=very useful)",
-        "fr": "Dans quelle mesure la navigation est-elle facile dans l’InfoBase du GC?"
+        "fr": "Dans quelle mesure la navigation est-elle facile dans l’InfoBase du GC? (1=difficile et 5=très facile)"
```
Comment (bill-lliu) [inline email_backend/templates/feedback.json:42]: I think this was mixed with the other french 1-5 scale

File: email_backend/templates/feedback.json [hunk 3]
```diff
@@ -81,7 +81,7 @@
       "form_type": "radio",
       "form_label": {
         "en": "To what extent did you find the spending information on GC InfoBase useful? (1=not at all easy and 5=very easy)",
-        "fr": "Dans quelle mesure avez-vous trouvé les renseignements financiers utiles dans l’InfoBase du GC?"
+        "fr": "Dans quelle mesure avez-vous trouvé les renseignements financiers utiles dans l’InfoBase du GC? (1=inutile et 5=très utile)"
```
Comment (bill-lliu) [inline email_backend/templates/feedback.json:84]: this is the other piece that was mixed

File: email_backend/templates/feedback.json (hunk 2 above)
Comment (Stephen-ONeil) [reply email_backend/templates/feedback.json]: Woops

**Comment by Stephen-ONeil** [comment]
All good aside from the French mix-ups?

**Comment by bill-lliu** [comment]
> All good aside from the French mix-ups?

Seems so yep, think I'm gonna merge and rebase the mongo branch now

**Pull request merged**